api/all/src/main/java/io/opentelemetry/api/logs/LogEmitterBuilder.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.logs;
+
+import io.opentelemetry.api.common.Attributes;
+
+public interface LogEmitterBuilder {
```

Comment:
Here we can also add another option to control whether the span context is added to the log record or no.